protocol/synthetix/contracts/interfaces/IMulticallModule.sol

```diff
@@ -11,7 +11,7 @@ interface IMulticallModule {
      * @param data Array of calldata objects, one for each function that is to be called in the system.
      * @return results Array of each `delegatecall`'s response corresponding to the incoming calldata array.
      */
-    function multicall(bytes[] calldata data) external payable returns (bytes[] memory results);
+    function multicall(bytes[] calldata data) external returns (bytes[] memory results);
```

Comment:
What was the reason for this change? Will I still be able to call payable methods when the `multicall` isn't payable?